### PR TITLE
feat: add language button

### PR DESCRIPTION
**Pull request by mikelxk**
refactor: use composition api

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -243,39 +241,35 @@
...
-  data() {
-    return {
-      isChrome: true
-    };
-  }
-};
+<script setup>
+import { useI18n } from 'vue-i18n';
+const { t, locale } = useI18n();
+const isChrome = false;
```
Comment (listen1) [inline src/views/Settings.vue:272]: extension env `isChrome` is true

File: vue.config.js [hunk 2]
```diff
@@ -11,12 +11,21 @@ module.exports = {
...
       .use('i18n')
-      .loader('@intlify/vue-i18n-loader');
+      .loader('@intlify/vue-i18n-loader')
+      .end();
+    config.module
+      .rule('vue')
+      .use('vue-loader')
+      .tap((options) => ({
+        ...options,
+        refSugar: true
```
Comment (listen1) [inline vue.config.js:27]: ref sugar seems not working. $ref is not defined is reported.

Comment (mikelxk) [reply vue.config.js:27]: ref sugar is working on my computer. Maybe you have to run yarn again to install the latest deps

**Comment by listen1** [comment]
ready to merge, really big pr, thanks for your work.

next time, please seperate pr by features, thanks.

**Pull request merged**